contracts/feature-tests/abi-tester/src/abi_tester.rs

```diff
@@ -16,6 +16,9 @@ use only_nested::*;
 /// Note: any change in this contract must also be reflected in `abi_test_expected.abi.json`,
 /// including Rust docs.
 #[multiversx_sc::contract]
+#[esdt_attribute("TICKER-1", BigUint)]
+#[esdt_attribute("TICKER-2", ManagedBuffer)]
+#[esdt_attribute("TICKER-3", u32)]
```

Comment:
Two more things: 
- also add a ticker with a structure/enum. You already have `AbiManagedType` and `AbiEnum` in the contract 
- create a structure `OnlyShowsUpInEsdtAttributes`, doesn't need to contain any fields, the point of it is, we want to see if all structures in esdt attributes get added to the types section of the ABI correctly.